Source/Particles/Filter/FilterFunctors.H

```diff
@@ -90,6 +93,8 @@ struct ParserFilter
 {
     /** constructor
      * \param a_is_active whether the test is active
+     * \param a_filter_parser ...?
+     * \param a_mass ...?
```

Comment:
@WeiqunZhang maybe? :)

Reply:
I think this was first implemented by @Yin-YinjianZhao .   Maybe Yin can take this?  I am not exactly sure what's `mass`.

Reply:
@WeiqunZhang @ax3l I actually don't know what it is neither, I don't think that was implemented by me.
But looking back the history, maybe @atmyers knows?
https://github.com/ECP-WarpX/WarpX/commit/3af650078efea347ef4522060ef8ca88b2e1e17d#diff-4c9a1b4d7c4f78632bd061ea54792a1e3e33e9bdad7d591ee0faf9f5d956571a

Reply:
Thanks for tracking  this! So `a_mass` is clear, i.e., `mass of the particle species`.
We just need a good final doc string for the `a_filter_parser` that is more descriptive than "this is a filter" :)